api-documentation.md

```diff
@@ -407,7 +407,10 @@ You can register and listen for two server-to-server callback events:
 In the following [link](https://www.pollfish.com/docs/s2s) you can find
 a step by step guide on how to register and receive these callbacks.
 
-Note: If you want to pass and receive your app\'s api key (param 35)
+Note 1: If you want to receive a value that you will sent on device_register
+and get back on S2S callback you can use the "click_id" parameter.
```

Comment:
If you want to pass a pram during intiialization and receive it back through s2s callback, you can use the "click_id" param